src/components/vm/nics/vmNicsCard.jsx

```diff
@@ -105,6 +107,95 @@ VmNetworkActions.propTypes = {
     networks: PropTypes.array.isRequired,
 };
 
+const NetworkSource = ({ network, networkId, vm, hostDevices }) => {
+    const id = vmId(vm.name);
+    const checkDeviceAvailability = (network) => {
+        for (const i in hostDevices) {
+            if (hostDevices[i].valid && hostDevices[i].Interface == network) {
+                return true;
+            }
+        }
+        return false;
+    };
+
+    const sourceJump = (source) => {
+        return () => {
+            if (source !== null && checkDeviceAvailability(source)) {
+                cockpit.jump(`/network#/${source}`, cockpit.transport.host);
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1077-20230626-124802-255e145d-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)